index.js

```diff
@@ -28,7 +28,13 @@ exports.open = function(filepath, opts, cb) {
     }
 
     if (opts && !!opts.watchForUpdates) {
+      if (opts.watchForUpdatesHook && typeof opts.watchForUpdatesHook != 'function') {
+        throw new Error('opts.watchForUpdatesHook should be a function');
+      }
       fs.watch(filepath, function() {
+        if (opts.watchForUpdatesHook) {
+          opts.watchForUpdatesHook();
```

Comment:
Should it happen AFTER the reload? Like around L40?

Reply:
Oh, sure you are right!